docs/requirements.txt

```diff
@@ -3,4 +3,4 @@ mkdocs-techdocs-core
 mkdocs-include-markdown-plugin
 mkdocs-include-dir-to-nav
 mkdocs-awesome-pages-plugin
-git+https://github.com/Andre601/mkdocs-pagenav-generator
+git+https://github.com/Andre601/mkdocs-pagenav-generator
```

Comment:
Okay, I don't even know what the diff is meant to be here :exploding_head: Think GitHub is lying to me about there being a change